.github/workflows/main.yml

```diff
@@ -77,15 +77,15 @@ jobs:
 #  Threadsafe runs
           - name: "Windows TS MSVC"
             artifact: "Windows-MSVCTS.tar.xz"
-            os: windows-2016
+            os: windows-latest
```

Comment:
Allen,  
windows 2022 is available will become the latest some time and it will have the VS 2022 environment.  Will the 2019 generator string still work with VS 2022?
If not should we hard code here for windows 2019 + VS 2019

Reply:
Agree. I should have looked. Will also need to change the version of cmake and generator when 2022 becomes latest.

Reply:
Updated to specify each VS available.